panos/network.py

```diff
@@ -1772,6 +1776,26 @@ def _setup(self):
             VersionedParamPath("metric", default=10, vartype="int", path="metric")
         )
 
+        params.append(
+            VersionedParamPath(
+                "enable_path_monitor", path="/path-monitor/enable", vartype="yesno"
+            )
+        )
+
+        params.append(
+            VersionedParamPath(
+                "failure_condition",
+                values=("all", "any"),
+                path="/path-monitor/failure-condition",
+            )
+        )
+
+        params.append(
+            VersionedParamPath(
+                "preemptive_holdtime", vartype="int", path="/path-monitor/hold-time"
```

Comment:
Please change `preemptive_holdtime` to `preemptive_hold_time` (and in the docstring as well).